docs: Include assigned user mode ALL (#1473)
dhis2/dhis2-docs: src/developer/web-api/tracker.md

| CURRENT | Includes events assigned to the current logged in user. |
| PROVIDED | Includes events assigned to the user provided in the request. |
| NONE | Includes unassigned events only. |
| ANY | Includes all assigned events, regardless of who they are assigned to, as long as they are assigned to someone. |

> | ALL | Includes all events irrespective of whether a user is assigned. This is the default mode. |

Reviewer: Can you please add to the request param table that `ALL` is the default mode.

Author: It's there already, you mean [here](https://github.com/dhis2/dhis2-docs/pull/1473/files#diff-4b5dabc4706dd8ae8c4c71cc43b6c8d08fa520c861a63cae3908ac407f201fffR1794) right?

| PROVIDED | Includes events assigned to the user provided in the request. |
| NONE | Includes unassigned events only. |

> | ANY | Includes all assigned events, doesn't matter who are they assigned to as long as they assigned to someone. |

Reviewer: ```suggestion
| ANY | Includes all assigned events, regardless of who they are assigned to, as long as they are assigned to someone. |
```

> | ALL | Includes all events irrespective of whether a user is assigned. If no mode is specified, this one is used by default.|

Reviewer: ```suggestion
| ALL | Includes all events irrespective of whether a user is assigned. This is the default mode.|
```

We usually describe the default in the request parameter table above. I think its missing above.
